### PR TITLE
Bugfix/cc 6123 add jinja2 tempates

### PR DESCRIPTION
**Pull request by jacek-klapacz**
__Story / Bug id:__ https://bugzilla.stxnext.pl/show_bug.cgi?id=6213
__Project / App:__ wdrozenie
__Reference person:__ @JacekKlapacz
__Description:__

add jinja2 templates to code, remove static html pages 

__Results:__
pep8: empty

Report
======
189 statements analysed.

Statistics by type
------------------

+---------+-------+-----------+-----------+------------+---------+
|type     |number |old number |difference |%documented |%badname |
+=========+=======+===========+===========+============+=========+
|module   |6      |6          |=          |100.00      |0.00     |
+---------+-------+-----------+-----------+------------+---------+
|class    |2      |2          |=          |100.00      |0.00     |
+---------+-------+-----------+-----------+------------+---------+
|method   |16     |16         |=          |100.00      |43.75    |
+---------+-------+-----------+-----------+------------+---------+
|function |17     |17         |=          |100.00      |5.88     |
+---------+-------+-----------+-----------+------------+---------+

External dependencies
---------------------
::

    flask (presence_analyzer.main,presence_analyzer.views,presence_analyzer.utils)
    presence_analyzer 
      \-main (presence_analyzer.tests,presence_analyzer.views,presence_analyzer,presence_analyzer.utils)
      \-utils (presence_analyzer.tests,presence_analyzer.views)
      \-views (presence_analyzer.tests,presence_analyzer)

Raw metrics
-----------

+----------+-------+------+---------+-----------+
|type      |number |%     |previous |difference |
+==========+=======+======+=========+===========+
|code      |241    |52.62 |241      |=          |
+----------+-------+------+---------+-----------+
|docstring |138    |30.13 |138      |=          |
+----------+-------+------+---------+-----------+
|comment   |7      |1.53  |7        |=          |
+----------+-------+------+---------+-----------+
|empty     |72     |15.72 |72       |=          |
+----------+-------+------+---------+-----------+

Duplication
-----------

+-------------------------+------+---------+-----------+
|                         |now   |previous |difference |
+=========================+======+=========+===========+
|nb duplicated lines      |0     |0        |=          |
+-------------------------+------+---------+-----------+
|percent duplicated lines |0.000 |0.000    |=          |
+-------------------------+------+---------+-----------+

Messages by category
--------------------

+-----------+-------+---------+-----------+
|type       |number |previous |difference |
+===========+=======+=========+===========+
|convention |8      |8        |=          |
+-----------+-------+---------+-----------+
|refactor   |0      |0        |=          |
+-----------+-------+---------+-----------+
|warning    |1      |1        |=          |
+-----------+-------+---------+-----------+
|error      |0      |0        |=          |
+-----------+-------+---------+-----------+

% errors / warnings by module
-----------------------------

+------------------------+------+--------+---------+-----------+
|module                  |error |warning |refactor |convention |
+========================+======+========+=========+===========+
|presence_analyzer.tests |0.00  |100.00  |0.00     |87.50      |
+------------------------+------+--------+---------+-----------+
|presence_analyzer.views |0.00  |0.00    |0.00     |12.50      |
+------------------------+------+--------+---------+-----------+

Messages
--------

+-----------------+------------+
|message id       |occurrences |
+=================+============+
|invalid-name     |8           |
+-----------------+------------+
|locally-disabled |5           |
+-----------------+------------+
|unused-import    |1           |
+-----------------+------------+
|file-ignored     |1           |
+-----------------+------------+

Global evaluation
-----------------
Your code has been rated at 9.52/10 (previous run: 9.52/10, +0.00)

__Migrations:__
N/A

__Rollback:__
N/A

__New imports / dependencies:__
N/A

__Unit Tests:__
Test parsing of CSV file. ... ok
Test mean presence time of given user. ... ok
Testing wrong user id. ... ok
Test mean start & end time of given user grouped by weekday ... ok
Testing wrong user id. ... ok
Test presence time of given user. ... ok
Testing wrong user id. ... ok
Test users listing. ... ok
Test main page redirect. ... ok
Test template: mean_time_weekday.html ... ok
Test template: presence_start_end.html ... ok
Test template: presence_weekday.html ... ok

----------------------------------------------------------------------
Ran 12 tests in 0.076s

### CODE REVIEW

File: src/presence_analyzer/static/js/base.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+function parseInterval(value) {
+    var result = new Date(1,1,1);
+    result.setMilliseconds(value*1000);
+    return result;
+}
+
+(function($) {
+    $(document).ready(function(){
+        var loading = $('#loading');
+        $.getJSON("/api/v1/users", function(result) {
```
Comment (jacek-klapacz) [inline src/presence_analyzer/static/js/base.js]: i have no idea how use here url_for without moving static file to templates directory. Any ideas?

Comment (tomaszwozniak) [inline src/presence_analyzer/static/js/base.js]: jest ok

File: src/presence_analyzer/templates/presence_weekday.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+	        chart_div.show();
+	        loading.hide();
+	        var chart = new google.visualization.PieChart(chart_div[0]);
+	        chart.draw(data, options);
+	    });
+	}
+
+	google.load("visualization", "1", {packages:["corechart"], 'language': 'en'});
+{% endblock %}
+
```
Comment (tomaszwozniak) [inline src/presence_analyzer/templates/presence_weekday.html]: extra blank lines

File: src/presence_analyzer/views.py [hunk 3]
```diff
@@ -34,9 +60,10 @@ def users_view():
     ]
 
 
+@app.route('/api/v1/mean_time_weekday/')
 @app.route('/api/v1/mean_time_weekday/<int:user_id>', methods=['GET'])
 @utils.jsonify
-def mean_time_weekday_view(user_id):
+def mean_time_weekday_json_view(user_id=0):
```
Comment (tomaszwozniak) [inline src/presence_analyzer/views.py]: user_id=None

File: src/presence_analyzer/views.py [hunk 4]
```diff
@@ -54,9 +81,10 @@ def mean_time_weekday_view(user_id):
     return result
 
 
+@app.route('/api/v1/presence_weekday/')
 @app.route('/api/v1/presence_weekday/<int:user_id>', methods=['GET'])
 @utils.jsonify
-def presence_weekday_view(user_id):
+def presence_weekday_json_view(user_id=0):
```
Comment (tomaszwozniak) [inline src/presence_analyzer/views.py]: ditto

File: src/presence_analyzer/views.py [hunk 5]
```diff
@@ -75,9 +103,10 @@ def presence_weekday_view(user_id):
     return result
 
 
+@app.route('/api/v1/presence_start_end/')
 @app.route('/api/v1/presence_start_end/<int:user_id>', methods=['GET'])
 @utils.jsonify
-def presence_start_end_view(user_id):
+def presence_start_end_json_view(user_id=0):
```
Comment (tomaszwozniak) [inline src/presence_analyzer/views.py]: ditto

**Comment by tomaszwozniak** [comment]
minor comments

**Comment by jacek-klapacz** [comment]
__Story / Bug id:__ https://bugzilla.stxnext.pl/show_bug.cgi?id=6213
__Project / App:__ wdrozenie
__Reference person:__ @JacekKlapacz
__Description:__

add jinja2 templates, code cosmeticks

__Results:__
pep8: empty

Report
======
189 statements analysed.

Statistics by type
------------------

+---------+-------+-----------+-----------+------------+---------+
|type     |number |old number |difference |%documented |%badname |
+=========+=======+===========+===========+============+=========+
|module   |6      |6          |=          |100.00      |0.00     |
+---------+-------+-----------+-----------+------------+---------+
|class    |2      |2          |=          |100.00      |0.00     |
+---------+-------+-----------+-----------+------------+---------+
|method   |16     |16         |=          |100.00      |43.75    |
+---------+-------+-----------+-----------+------------+---------+
|function |17     |17         |=          |100.00      |5.88     |
+---------+-------+-----------+-----------+------------+---------+



External dependencies
---------------------
::

    flask (presence_analyzer.main,presence_analyzer.views,presence_analyzer.utils)
    presence_analyzer 
      \-main (presence_analyzer.tests,presence_analyzer.views,presence_analyzer,presence_analyzer.utils)
      \-utils (presence_analyzer.tests,presence_analyzer.views)
      \-views (presence_analyzer.tests,presence_analyzer)



Raw metrics
-----------

+----------+-------+------+---------+-----------+
|type      |number |%     |previous |difference |
+==========+=======+======+=========+===========+
|code      |241    |52.62 |241      |=          |
+----------+-------+------+---------+-----------+
|docstring |138    |30.13 |138      |=          |
+----------+-------+------+---------+-----------+
|comment   |7      |1.53  |7        |=          |
+----------+-------+------+---------+-----------+
|empty     |72     |15.72 |72       |=          |
+----------+-------+------+---------+-----------+



Duplication
-----------

+-------------------------+------+---------+-----------+
|                         |now   |previous |difference |
+=========================+======+=========+===========+
|nb duplicated lines      |0     |0        |=          |
+-------------------------+------+---------+-----------+
|percent duplicated lines |0.000 |0.000    |=          |
+-------------------------+------+---------+-----------+



Messages by category
--------------------

+-----------+-------+---------+-----------+
|type       |number |previous |difference |
+===========+=======+=========+===========+
|convention |8      |8        |=          |
+-----------+-------+---------+-----------+
|refactor   |0      |0        |=          |
+-----------+-------+---------+-----------+
|warning    |1      |1        |=          |
+-----------+-------+---------+-----------+
|error      |0      |0        |=          |
+-----------+-------+---------+-----------+



% errors / warnings by module
-----------------------------

+------------------------+------+--------+---------+-----------+
|module                  |error |warning |refactor |convention |
+========================+======+========+=========+===========+
|presence_analyzer.tests |0.00  |100.00  |0.00     |87.50      |
+------------------------+------+--------+---------+-----------+
|presence_analyzer.views |0.00  |0.00    |0.00     |12.50      |
+------------------------+------+--------+---------+-----------+



Messages
--------

+-----------------+------------+
|message id       |occurrences |
+=================+============+
|invalid-name     |8           |
+-----------------+------------+
|locally-disabled |5           |
+-----------------+------------+
|unused-import    |1           |
+-----------------+------------+
|file-ignored     |1           |
+-----------------+------------+



Global evaluation
-----------------
Your code has been rated at 9.52/10 (previous run: 9.52/10, +0.00)



__Migrations:__
N/A

__Rollback:__
N/A

__New imports / dependencies:__
N/A

__Unit Tests:__
Test parsing of CSV file. ... ok
Test mean presence time of given user. ... ok
Testing wrong user id. ... ok
Test mean start & end time of given user grouped by weekday ... ok
Testing wrong user id. ... ok
Test presence time of given user. ... ok
Testing wrong user id. ... ok
Test users listing. ... ok
Test main page redirect. ... ok
Test template: mean_time_weekday.html ... ok
Test template: presence_start_end.html ... ok
Test template: presence_weekday.html ... ok

----------------------------------------------------------------------
Ran 12 tests in 0.097s

**Pull request merged**